rackcity/utils/rackcity_utils.py

```diff
@@ -179,26 +175,6 @@ def validate_location_modification(data, existing_asset, user, change_plan=None)
         raise error
 
 
-def get_change_plan(change_plan_id):
```

Comment:
5. Moved some stuff from `rackcity_utils.py` to more specific/relevant utils folders